tldw_Server_API/tests/MediaIngestion_NEW/unit/test_process_batch_media_precheck_regressions.py

```
@@ -17,6 +17,26 @@ def fetchone(self):
         return self._row
 
 
+class _FakePrecheckDb:
+    def __init__(self, *, row=None, error: Exception | None = None, has_source_hash: bool = False):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)